docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -15,15 +15,18 @@ ifdef::disconnected[]
 You can install the {PluginName} in a disconnected network environment.
 endif::[]
 
-The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2021-03 and Red Hat CodeReady Studio 12.19.1.
+The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2021-09 and Red Hat CodeReady Studio 12.21.0.GA.
 
 [discrete]
 include::jdk-hardware-mac-prerequisites.adoc[leveloffset=+2]
 
-* link:{CodeReadyStudioDownloadPageURL}[Red Hat CodeReady Studio] _or_ link:https://www.eclipse.org/downloads/packages/release/2021-03/r/eclipse-ide-java-developers[Eclipse IDE for Java Enterprise Developers 2021-03]. If you are installing Eclipse, be sure to download Eclipse IDE for Java Enterprise Developers 2021-03, not 2021-06.
-* JBoss Tools, installed with the link:https://www.eclipse.org/mpc/[Eclipse Marketplace Client].
-
+* link:{CodeReadyStudioDownloadPageURL}[Red Hat CodeReady Studio] _or_ link:https://www.eclipse.org/downloads/packages/release/2021-09/r/eclipse-ide-java-developers[Eclipse IDE for Java Enterprise Developers 2021-09]
+* JBoss Tools, installed with the link:https://www.eclipse.org/mpc/[Eclipse Marketplace Client]
+* link:http://download.eclipse.org/mylyn/releases/latest[Mylyn SDK and frameworks] installed
```

Comment:
Maybe change "installed" to ", installed with Eclipse" so that it is consistent with JBoss Tools.